### PR TITLE
Add exception for `tar-content` in `snapcraft help`

### PR DESCRIPTION
**Pull request by tsimonq2**
(This fixes bug [1604312](https://pad.lv/1604312))

This ensures that `snapcraft help tar-content` prints:
```
The plugin has no documentation
```
Instead of:
```
The plugin does not exist. Run `snapcraft list-plugins` to see the available plugins.
```

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: snapcraft/_help.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def _topic_help(module_name, devel):
 
 
 def _module_help(module_name, devel):
+    if module_name == "tar-content":
```
Comment (sergiusens) [inline snapcraft/_help.py]: instead of this, can you replace all `-` to `_` like we do in the plugin loader?

**Comment by sergiusens** [comment]
looks good thanks, kudos for the test. I would like to however be future proof and make the replacement generic as I mentioned in the in-code comment

**Comment by tsimonq2** [comment]
@sergiusens done

File: snapcraft/_help.py [hunk 2]
```diff
@@ -44,6 +44,8 @@ def _topic_help(module_name, devel):
 
 
 def _module_help(module_name, devel):
+    if "-" in module_name:
+        module_name.replace("-", "_")
```
Comment (sergiusens) [inline snapcraft/_help.py]: but but ' instead of " :smiley:

Comment (sergiusens) [inline snapcraft/_help.py]: and no need for the conditional btw.

Comment (tsimonq2) [inline snapcraft/_help.py]: Done

**Comment by sergiusens** [comment]
```
======================================================================
ERROR: test_tar_content_never_nonexistent (snapcraft.tests.test_commands_help.HelpCommandTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/snapcore/snapcraft/snapcraft/_help.py", line 50, in _module_help
    'snapcraft.plugins.{}'.format(module_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'snapcraft.plugins.tar-content'
```

**Comment by sergiusens** [comment]
Closeing due to inactivity

**Comment by telelaci** [comment]
Wouldn't be easier to write a few line documentation, then print "No document, go away"
It can't be so sophisticated plugin.
jeez.

**Pull request closed without merging**